task/script.js

```diff
@@ -221,6 +223,21 @@ taskForm.onsubmit = async (e) => {
     delete dataToBeSent.links;
   }
 
+  dataToBeSent.dependsOn = dataToBeSent.dependsOn.filter(
+    (dependOn) => dependOn,
+  );
+  console.log(dependsOn);
+  if (dataToBeSent.dependsOn.length !== 0) {
+    dataToBeSent.dependsOn = dataToBeSent.dependsOn[0].split(',');
+    dataToBeSent.dependsOn = dataToBeSent.dependsOn.filter(
+      (dependOn) => dependOn,
+    );
+    console.log('230', dependsOn);
```

Comment:
is it good to keep this console log in the client machine?

Reply:
removed